File: backend/DescarTec.Api/Models/Dto/SsoDto.cs
namespace DescarTec.Api.Models.Dto
{
    public class SsoDto
    {
        public string Access_token { get; set; }
        public DayOfWeek Expiration { get; set; }
        public UserBase User { get; set; }

        public SsoDto(string access_token, UserBase user)
        {
            this.Access_token = access_token;
            this.Expiration = DateTime.UtcNow.DayOfWeek;
            this.User = user;
        }
    }
}

File: frontend/webapp/src/app/models/dto/ssoDto.ts
import { User } from "../user";

export class SsoDto {
    access_token: string;
    expiration: string;
    user: User;
    
    constructor(access_token: string, expiration: string, user: User) {
		this.access_token = access_token;
		this.expiration = expiration;
		this.user = user;
	}
}